### PR TITLE
Scale metrics to appropriate dimension

### PR DESCRIPTION
**Pull request by shadowshot-x**
For some metrics (Eg:- storage.googleapis.com/google_egress_bandwidth), the value retrieved has different dimension as compared to the dimension of limit. This needs to be fixed as it can result in incorrect alerts being sent.

### CODE REVIEW

File: quota-scan/src/main/java/functions/eventpojos/MetricFix.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package functions.eventpojos;
+
+public class MetricFix {
```
Comment (shadowshot-x) [inline quota-scan/src/main/java/functions/eventpojos/MetricFix.java:16]: @anuradha-bajpai-google Can you suggest a better name for this class?

Comment (shadowshot-x) [reply quota-scan/src/main/java/functions/eventpojos/MetricFix.java:16]: cc: @bgood

**Comment by anuradha-bajpai-google** [comment]
Thank you for submitting the PR. I have added one comment. Please see.

**Comment by shadowshot-x** [comment]
> Thank you for submitting the PR. I have added one comment. Please see.

Hi @anuradha-bajpai-google . Thanks for replying. I don't see any additional comments on the PR. Can you help me with this?

**Pull request closed without merging**